apps/settings/elements/root.html

```diff
@@ -140,6 +140,14 @@ <h2 data-l10n-id="accounts"></h2>
             <small id="findmydevice-desc" class="findmydevice-desc menu-item-desc"></small>
           </a>
         </li>
+        <!--IFDEF_FIREFOX_SYNC
+        <li>
+          <a id="menuItem-fxsync" class="menu-item" data-icon="download" href="#fxsync" data-l10n-id="fxsyncSection" aria-describedby="fxsync-desc">
+            <span data-l10n-id="fxsync"></span>
+            <small id="fxsync-desc" class="fxsync-desc menu-item-desc"></small>
```

Comment:
I would prefer if this description was updated (disabled/enabled) when sync is on